frontend/assets/stylesheets/components/_icons.scss

```diff
@@ -68,6 +68,11 @@
    height: 13px;
 }
 
+.icon-inline--medium-small {
```

Comment:
Blerg, can we try 16px? Not sure about having a different sized icon modifier just for one case.